packages/flutter/ios/sentry_flutter/Sources/sentry_flutter/SentryFlutterPlugin.swift

```
@@ -623,6 +608,21 @@ public class SentryFlutterPlugin: NSObject, FlutterPlugin {
         return nil
   }
 
+  @objc public class func setUserAsBytes(_ userBytes: NSData?) {
+      guard let userBytes = userBytes else {
+          SentrySDK.setUser(nil)
+          return
+      }
+
+      guard let userString = String(data: userBytes as Data, encoding: .utf8),
+            let jsonData = userString.data(using: .utf8),
+            let userDict = try? JSONSerialization.jsonObject(with: jsonData) as? [String: Any] else {
```

We should catch the error and log if possible. now we silently drop any error here.